### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by casualdegenerate** [Fabricators-of-Create/Porting-Lib#37]
Log: https://pastebin.com/zDDQCgYh
I'm pulling my hair out trying to figure out why this is not working.

**Comment by TropheusJ** [comment]
porting lib 1.19.3 is incomplete and in the middle of refactors

**Comment by casualdegenerate** [comment]
(Warning: Alphabet Soup, you don't have to read this.)
Okay, because I was adding in more mods to my modpack, and I was confused on which mod was causing issues. And some mods used porting-lib, and for some reason the mod would not be removed from /mods/ so I had to delete it manually(very strange), but I was able to get my modpack to work. Left this up here so you can solve it. But it seems it's something you are working on. I don't know why, but ATLauncher(or a mod) attempted to download 1.19.3 version of the Porting-Lib.

TL; DR: Thanks on the fast response.
